### PR TITLE
Set terminationGracePeriodSeconds for HelmCharts

### PR DESCRIPTION
**Pull request by M1l1j0n**
### Proposed changes

We would like to override value of terminationGracePeriodSeconds parameter. Unfortunately pulumi kubernetes operator helm chart is not setting this value.

### Related issues (optional)

By default, the terminationGracePeriodSeconds parameter is not set, so the operator will take no action. If this parameter is set then the value will be overwritten.

impact/no-changelog-required

### CODE REVIEW

File: deploy/helm/pulumi-operator/values.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ deploymentStrategy: ""
 # -- Specifies the replica count for the deployment
 replicaCount: 1
 
+# -- Specifies termination grace period
+terminationGracePeriodSeconds:
```
Comment (dirien) [inline deploy/helm/pulumi-operator/values.yaml:8]: Can you enter a meaningful default value

**Review by dirien — CHANGES_REQUESTED** [review]
Hey @M1l1j0n,

thanks for the PR. Can you bump the minor version in the `Chart.yaml` and add `artifacthub.io/changes:` text?

File: deploy/helm/pulumi-operator/values.yaml (hunk 1 above)
Comment (M1l1j0n) [reply deploy/helm/pulumi-operator/values.yaml]: Done.

**Comment by M1l1j0n** [comment]
Done @dirien 

> Hey @M1l1j0n,
> 
> thanks for the PR. Can you bump the minor version in the `Chart.yaml` and add `artifacthub.io/changes:` text?

**Review by dirien — CHANGES_REQUESTED** [review]
@M1l1j0n, can you give me rights to push on your branch? I need to update the `README.md` 

Or could you call this command: `helm-docs -t README.md.gotmpl -o README.md -b for-the-badge` with the `helm-docs` cli?

**Comment by M1l1j0n** [comment]
@dirien I have called helm-docs command.

> @M1l1j0n, can you give me rights to push on your branch? I need to update the `README.md`
> 
> Or could you call this command: `helm-docs -t README.md.gotmpl -o README.md -b for-the-badge` with the `helm-docs` cli?

**Review by dirien — APPROVED** [review]
Hey @M1l1j0n,

thanks for your contribution and lgtm!

**Pull request merged**